.vscode/extensions.json

```diff
@@ -0,0 +1,13 @@
+{
```

Comment:
Why? It makes onboarding the new developer easier.
VSCode will recommend the developer to install the extensions which make the development easier. If the extensions are already installed, it will not bother the user.